binder/apt.txt

```diff
@@ -0,0 +1,6 @@
+libgl1
+libgl1-mesa-glx
+libosmesa6-dev
+gcc
+xvfb
+python3-opencv
```

Comment:
what for?

Reply:
Without opencv, I was gettting the following error:  `ImportError: libGL.so.1: cannot open shared object file: No such file or directory`

Reply:
hum, ok. GL is definitely a subset of opencv.